watertap/unit_models/tests/test_boron_removal.py

```diff
@@ -123,787 +81,193 @@ def scaling_setup(
-        assert isinstance(m.fs.unit.eq_water_dissociation, Constraint)
-        assert isinstance(m.fs.unit.eq_boron_dissociation, Constraint)
-
-    @pytest.mark.unit
-    def test_stats(self, base_boron_removal_model):
-        m = base_boron_removal_model
-
-        # Check to make sure we have the correct DOF and
-        #   check to make sure the units are correct
-        assert_units_consistent(m)
-        assert degrees_of_freedom(m) == 8
-
-        # set the variables
-        model_setup(m, chem_add=0)
-
-        assert degrees_of_freedom(m) == 0
-
-    @pytest.mark.unit
-    def test_scaling(self, base_boron_removal_model):
-        m = base_boron_removal_model
-
-        # Set some scaling factors and look for 'bad' scaling
-        scaling_setup(m)
-
-        # check that all variables have scaling factors
-        unscaled_var_list = list(iscale.unscaled_variables_generator(m))
-        assert len(unscaled_var_list) == 0
-
-        # check if any variables are badly scaled
-        badly_scaled_var_values = {
-            var.name: val
-            for (var, val) in iscale.badly_scaled_var_generator(
-                m, large=1e3, small=1e-3
-            )
-        }
-        assert not badly_scaled_var_values
-
-    @pytest.mark.component
-    def test_initialization(self, base_boron_removal_model):
-        m = base_boron_removal_model
-        initialization_tester(m)
-
-        # check to make sure DOF does not change
-        assert degrees_of_freedom(m) == 0
-
-    @pytest.mark.component
-    def test_solve(self, base_boron_removal_model):
-        m = base_boron_removal_model
-
-        # first, check to make sure that after initialized, the scaling is still good
-        badly_scaled_var_values = {
-            var.name: val
-            for (var, val) in iscale.badly_scaled_var_generator(
-                m, large=1e3, small=1e-3
-            )
-        }
-        assert not badly_scaled_var_values
-
-        # run solver and check for optimal solution
-        results = solver.solve(m)
-        assert_optimal_termination(results)
-
-    @pytest.mark.component
-    def test_solution(self, base_boron_removal_model):
-        m = base_boron_removal_model
-
-        assert value(
-            m.fs.unit.outlet.flow_mol_phase_comp[0, "Liq", m.fs.unit.boron_name_id]
-        ) == pytest.approx(1.20642e-4, rel=1e-4)
-        assert value(
-            m.fs.unit.outlet.flow_mol_phase_comp[0, "Liq", m.fs.unit.borate_name_id]
-        ) == pytest.approx(8.03579e-5, rel=1e-4)
-        assert value(m.fs.unit.outlet_pH()) == pytest.approx(9.0343, rel=1e-4)
-        assert value(m.fs.unit.outlet_pOH()) == pytest.approx(4.9621, rel=1e-4)
-
-
-# -----------------------------------------------------------------------------
-# Start test class
-class TestBoronRemoval_GenPropPack:
-    @pytest.fixture(scope="class")
-    def gen_boron_removal_model(self):
-        m = ConcreteModel()
-        m.fs = FlowsheetBlock(dynamic=False)
-
```

Comment:
Unless I missed it, this gen prop pack test is missing from the update version. Is there a reason this test should be left out?

Reply:
I had left it out initially because it had seemed redundant to test the unit model with more than 1 property model (and using a property model outside of watertap). However, if this is an important aspect for testing, I can add on these tests.

Reply:
I think it can be left out. I just wanted to make sure you had deleted it intentionally rather than accidentally leaving it out.